assets/app/styles/_core.less

```diff
@@ -219,6 +219,12 @@ html, body {
         text-align: right;
       }
     }
+    .form-group {
+      width: 44%;
```

Comment:
explain what this is doing and how to check it in the UI so if we need to adjust it we'll know where to look